% Original Template Author: Rishi Shah

% Document Configuration
\documentclass{resume}
\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry}
\newcommand{\tab}[1]{\hspace{.2667\textwidth}\rlap{#1}}
\newcommand{\itab}[1]{\hspace{0em}\rlap{#1}}
\name{Jack Gisby}
\address{\faGithub{\hspace{0.3mm} \href{https://github.com/jackgisby}{ github.com/jackgisby}} \hspace{0.3mm}  \faEnvelope{\hspace{0.3mm} j.gisby20@imperial.ac.uk}}
\address{\faMapMarker{\hspace{0.3mm} London, United Kingdom} \hspace{0.15mm} \faPhone{\hspace{0.15mm} (+44)7748 559964}}

% Document Content 
\begin{document}
\begin{rSection}{Research Interests}
\vspace{1pt plus 1pt}
Data scientist and computational biologist interested in data-driven research and the -Omics technologies. My PhD work involves integrating genomic, transcriptomic and proteomic datasets to identify molecular pathways that play a driving role in inflammatory disease. I am also interested in developing data processing and mining methods to facilitate the extraction of biological insights from -Omics datasets.

\end{rSection}
\begin{rSection}{Education}

\vspace{1pt plus 1pt}
{\bf Imperial College London} \hfill {\em July 2020 - August 2023} 
\\ PhD Candidate

\smallskip

{\bf University of Birmingham} \hfill {\em September 2016 - June 2020} 
\\ Biochemistry with Professional Placement (MSci) - First Class
\end{rSection}

% \begin{rSection}{Awards and grants}
% \end{rSection}

\begin{rSection}{Key Publications}

\vspace{1pt plus 1pt}
\textbf{Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death} \\
First author \hfill  \textit{eLife} 2021 - \href{https://doi.org/10.7554/eLife.64827}{doi:10.7554/eLife.64827}

\vspace{2pt plus 1pt minus 1pt}
\item Lead the analysis of a high-dimensional proteomics (Olink) dataset with a complex cohort consisting of repeated measurements at inconsistent time points.
\item Applied linear mixed models and joint models to identify key proteins that changed over time following COVID-19 symptom onset.
\item Utilised supervised learning algorithms to identify biomarkers of severe disease.\\

\textbf{Mendelian randomisation identifies alternative splicing of the FAS death receptor as a mediator of severe COVID-19} \\
Second author (equal contribution) \hfill  \textit{medRxiv} 2021 - \href{https://doi.org/10.1101/2021.04.01.21254789}{doi:10.1101/2021.04.01.21254789}

\vspace{2pt plus 1pt minus 1pt}
\item Developed a parallelised \href{https://github.com/jackgisby/mr-nextflow}{pipeline for two sample Mendelian Randomisation} and subsequently investigated COVID-19 susceptibility and severity using Olink \& SOMA pQTLs. 
\item Used fine mapping and colocalisation tools in tandem with public data repositories to investigate causal variants of disease. \\

\textbf{The widespread nature of Pack-TYPE transposons reveals their importance for plant genome evolution} \\
First author \hfill  \textit{bioRxiv} 2021 - \href{https://doi.org/10.1101/2021.06.18.448592}{doi:10.1101/2021.06.18.448592}

\vspace{2pt plus 1pt minus 1pt}
\item Designed an algorithm for the specific annotation of repetitive elements that capture chromosomal DNA; implemented this as an \href{https://bioconductor.org/packages/release/bioc/html/packFinder.html}{R package available as part of the Bioconductor project}.
\item Mined open source genetic data to demonstrate the abundance of these elements for multiple superfamilies and genomes. Found that recent insertions of these elements have impacted the evolution of genes. \\

\textbf{Plasma Lectin Pathway Complement Proteins in Patients With COVID-19 and Renal Disease} \\
Co-author \hfill  \textit{Frontiers in Immunology} 2021 - \href{https://doi.org/10.3389/fimmu.2021.671052}{doi:10.3389/fimmu.2021.671052}

\vspace{2pt plus 1pt minus 1pt}
\item Provided statistical support for the analysis of a repeated measures study design.
\item Generated data visualisations to investigate biological hypotheses. \\

\end{rSection}

\begin{rSection}{Conferences and presentations}

\item \textbf{Presentation - Biomarkers of the Future 2021} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death
\vspace{2pt plus 1pt minus 1pt}

\item \textbf{Poster - HUPO Reconnect 2021} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death
\vspace{2pt plus 1pt minus 1pt}

\item \textbf{Poster - UK-CIC Immunology 2021} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death
\vspace{2pt plus 1pt minus 1pt}

\item \textbf{Presentation - Longitudinal Studies 2021, Wellcome Genome Campus} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death
\vspace{2pt plus 1pt minus 1pt}

\end{rSection}

\begin{rSection}{Additional research experience}

\vspace{1pt plus 1pt}
\textbf{A Prior Knowledge-Based Computational Workflow for \textit{de novo} Structural Elucidation of Small Molecules in Mass Spectrometry Metabolomics} \\
Dissertation project, Dr Ralf Weber \hfill  \textit{October 2019 - June 2020}

\vspace{2pt plus 1pt minus 1pt}
\item Developed and optimised a Python package, \href{https://github.com/computational-metabolomics/metaboblend/tree/dev}{\textit{MetaboBlend}}, to propose candidate molecules using tandem mass spectra. 
\item Processed large, high-dimensional mass-spectrometry datasets for the validation of \href{https://github.com/computational-metabolomics/metaboblend/tree/dev}{\textit{MetaboBlend}}. 
\item Used Python, SQLite and Bash to implement the \href{https://github.com/computational-metabolomics/metaboblend/tree/dev}{\textit{MetaboBlend}} workflow on a Linux high-performance computing cluster for the generation of large structure databases.
\item Created a "metabolite-likeness" model based on a deep learning implementation of the one-class support vector data description algorithm, adapted to model metabolites with a transformer architecture. \\

\textbf{Development of an Immunoturbidimetric Assay for Serum Amyloid A for an Automated Clinical Analyser} \\
Industrial Placement, The Binding Site \hfill  \textit{August 2018 - August 2019}

\vspace{2pt plus 1pt minus 1pt}
\item Developed assays for inflammatory biomarkers, such as Serum Amyloid A, for use in clinical laboratories.
\item Interacted with, and regularly delivered presentations to, researchers from diverse fields and non-scientific staff to ensure the timely completion of my development project. 
\item Took responsibility for planning and carrying out experiments and statistical analyses in a research environment.

\end{rSection}

\begin{rSection}{Teaching experience}

\item \textbf{Lecture:} RNA-seq in Molecular Epidemiology - Developed a two-part lecture explaining the basics of RNA sequencing and its application to investigate genetic variants. Set homework and final exam questions for the module.
\vspace{2pt plus 1pt minus 1pt}

\item \textbf{Supervision:} Co-supervisor of a medical undergraduate - On-going project using Olink proteomics to investigate Lupus.

\vspace{2pt plus 1pt minus 1pt}

\item \textbf{Volunteer:} Code Club - Developed my teaching abilities and provided help with coding skills to 9-13 year-olds as part of the Code Club voluntary initiative.
\vspace{2pt plus 1pt minus 1pt}

\end{rSection}
\begin{rSection}{Specific Computational Skills}

\begin{tabular}{ @{} >{\bfseries}l @{\hspace{3ex}} l }
Operating Systems \ & Windows, Ubuntu \vspace{2pt plus 1pt minus 1pt} \\
Programming Languages \ & Python, R, SQLite, Bash, working knowledge of C++ \& Julia \vspace{2pt plus 1pt minus 1pt} \\
Other Tools \ & Git, Linux HPC, Conda, Docker, Nextflow \vspace{2pt plus 1pt minus 1pt}\\
General Software \ & MS Office, \LaTeX  \vspace{2pt plus 1pt minus 1pt} \\
Data types \ & Olink \& SOMA proteomics, RNA-seq, MS metabolomics, \\
    \ & whole genome sequences, GWAS \vspace{2pt plus 1pt minus 1pt} \\ 
Bioinformatics/Statistics \ & Package development in R and Python, Linear and mixed models, \\
    \ & Mendelian randomisation \& colocalisation, Joint models, \\
    \ & Supervised learning (scikit-learn, tensorflow, pytorch), WGCNA \\
\end{tabular}

% \begin{rSection}{Additional publications}
% \end{rSection}

\end{rSection}

\begin{rSection}{References}

\begin{center}
\begin{tabular}{c@{\hskip 0.75in}c} 

 Dr James Peters (PhD Supervisor) & Dr Marco Catoni (Undergraduate Project) \\ 
 Clinical Reader in Rheumatology & Lecturer in Plant Biology \\ 
 Faculty of Medicine & School of Biosciences \\ 
 Imperial College London & University of Birmingham \\ 
 Hammersmith & Edgbaston \\ 
 W12 0NN & B15 2TT \\ 
 j.peters@imperial.ac.uk & m.catoni@bham.ac.uk \\ 
\end{tabular}
\end{center}

\end{rSection}
\end{document}
