% Original Template Author: Rishi Shah

% Document Configuration
\documentclass{resume}

\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry}
\usepackage{orcidlink}

\newcommand{\tab}[1]{\hspace{.2667\textwidth}\rlap{#1}}
\newcommand{\itab}[1]{\hspace{0em}\rlap{#1}}

%----------------------- Title
\name{Jack Gisby}
\address{\faEnvelope{\hspace{0.3mm} j.gisby20@imperial.ac.uk} \hspace{0.15mm} \faPhone{\hspace{0.15mm} (+44)7748 559964} \hspace{0.15mm}}
\address{\faGithub{\hspace{0.3mm} \href{https://github.com/jackgisby}{ github.com/jackgisby}} \hspace{0.15mm} \orcidlink{https://orcid.org/0000-0003-0511-8123}{\hspace{0.3mm} \href{{https://orcid.org/0000-0003-0511-8123}}{orcid.org/0000-0003-0511-8123}}}
% \address{\faMapMarker{\hspace{0.3mm} London, United Kingdom}}

% Document Content 
\begin{document}
% \begin{rSection}{Research Interests}
% \vspace{1pt plus 1pt}
% Data scientist and computational biologist interested in data-driven research and the -Omics technologies. Integrated genomic, transcriptomic and proteomic datasets to identify molecular pathways that play a driving role in inflammatory disease. Developed data processing and mining methods to facilitate the extraction of biological insights from large datasets. Experienced in both carrying out independent research and collaborating on interdisciplinary projects.

% \end{rSection}
%----------------------- Education
\begin{rSection}{Education}

\vspace{1pt plus 1pt}
{\bf Imperial College London} \hfill {\em August 2020 -- July 2023} 
\\ PhD Candidate
\item Thesis: Using multi-omics to investigate the host response in COVID-19, supervised by Dr James Peters.

% \smallskip

{\bf University of Birmingham} \hfill {\em September 2016 -- June 2020} 
\\ Biochemistry with Professional Placement (MSci) - First Class
\item Dissertation: A prior knowledge-based computational workflow for \textit{de novo} structural elucidation of small molecules in mass spectrometry metabolomics, supervised by Dr Ralf Weber.

\end{rSection}
%----------------------- Research Experience
\begin{rSection}{Research experience}

\vspace{1pt plus 1pt}

% phd
\textbf{PhD Research} \hfill \textit{August 2020 -- July 2023} \\
Immunology and Inflammation, Imperial College London

\vspace{2pt plus 1pt minus 1pt}
\item Integrated proteomic, transcriptomic and genetic data to investigate the pathology of severe COVID-19.
\item Applied longitudinal models, network analyses and machine learning to high-dimensional datasets.
\item Employed Mendelian randomisation and colocalisation to identify molecules that cause severe disease.
\item Used and developed Nextflow \& HPC-based pipelines to process sequence and variant data.
\item Disseminated research findings through peer-reviewed publications and conference presentations. 

% transposons project
\textbf{Research volunteer} \hfill \textit{July 2019 -- July 2020} \\
School of Biosciences, University of Birmingham

\vspace{2pt plus 1pt minus 1pt}
\item Developed an R/Bioconductor package for \href{https://bioconductor.org/packages/release/bioc/html/packFinder.html}{annotating rare transposons in genome sequences \faGithub}.
\item Mined genome sequence data to uncover the impact of DNA transposons on the evolution of host genes.

% msc dissertation
\textbf{MSci Research} \hfill \textit{October 2019 -- May 2020} \\
School of Biosciences, University of Birmingham

\vspace{2pt plus 1pt minus 1pt}
\item Developed a \href{https://github.com/jackgisby/metaboblend}{Python package for improved annotation of LC-MS metabolomics spectra \faGithub}.
\item Created a \href{https://github.com/jackgisby/deepmet}{deep learning-based anomaly detection model \faGithub} that classifies molecular structures.
\item Adhered to software engineering best practices, including version control and automated testing.

% the binding site
\textbf{Industrial placement} \hfill \textit{August 2018 -- July 2019} \\
The Binding Site, Birmingham

\vspace{2pt plus 1pt minus 1pt}
\item Took responsibility for the timely delivery of assay development projects and statistical reports.
\item Communicated with, and delivered presentations to, a wide variety of interdisciplinary staff.

\end{rSection}
%----------------------- Teaching experience
\begin{rSection}{Teaching experience}

% msc immunology workshops
\textbf{Graduate Teaching Assistant} \hfill  \textit{December 2022 - January 2023} \\
Immunology MSc Programme, Imperial College London
\vspace{2pt plus 1pt minus 1pt}
\item Aided in the development and delivery of a series of "Introduction to R programming" workshops.

% Lectures
\textbf{Lecturing} \hfill  \textit{September 2021 -- December 2022} \\
Molecular Epidemiology MSc Module, Imperial College London
\vspace{2pt plus 1pt minus 1pt}
\item Developed and delivered lectures for RNA sequencing in the context of investigating genetic variation. 
\item Set homework and reading, in addition to final exam questions and marking schemes.

\pagebreak

% recode project
\textbf{Graduate Teaching Assistant} \hfill  \textit{May 2022 -- August 2022} \\
Research Computing, Imperial College London
\vspace{2pt plus 1pt minus 1pt}
\item Developed a \href{https://github.com/ImperialCollegeLondon/ReCoDE_rnaseq_pipeline}{parallelised pipeline for processing RNA-seq data \faGithub} using Docker and Nextflow. 
\item Created teaching materials demonstrating best practices for building data pipelines.

% Supervision
\textbf{Mentoring} \hfill  \textit{July 2021 -- August 2022} \\
Immunology and Inflammation, Imperial College London
\vspace{2pt plus 1pt minus 1pt}
\item Co-supervised an undergraduate research project utilising proteomics to investigate Lupus.

\end{rSection}
%----------------------- Skills
\begin{rSection}{Skills}

\begin{tabular}{ @{} >{\bfseries}l @{\hspace{3ex}} l }
Languages \ & Python, R, Bash, SQL \\ % , exposure to C++, Julia \vspace{2pt plus 1pt minus 1pt}
Tools \ & Git/GitHub, Docker, Conda, Nextflow, Airflow, \LaTeX \vspace{2pt plus 1pt minus 1pt} \\
Compute \ & High performance computing clusters, Google Cloud Platform \vspace{2pt plus 1pt minus 1pt} \\
Data \ & Transcriptomics (bulk \& single-cell RNA-seq), proteomics (Olink \& SomaLogic assays), \\
     \ & genomics (sequences and summary), ontologies and clinical data \vspace{2pt plus 1pt minus 1pt} \\
Statistics \ & Linear \& mixed models, joint models, Mendelian randomisation \& colocalisation, \\
           \ & supervised learning (caret, scikit-learn, pytorch), network analysis (WGCNA) \vspace{2pt plus 1pt minus 1pt} \\
Software \ & Package development (R, Python), containerisation (Docker, Singularity),   \\
         \ & continuous integration (unittest, testthat), version control (Git, GitHub) \\
\end{tabular}

% \vspace{5pt}

\end{rSection}
%----------------------- Publications
\begin{rSection}{First author publications}

\vspace{1pt plus 1pt}

% LRRC15 multi-omics
\item \textbf{Jack S. Gisby}\textsuperscript{\textdagger}, Norzawani B. Buang\textsuperscript{\textdagger}, [...], David C. Thomas\textsuperscript{\textdagger}, James E. Peters\textsuperscript{\textdagger}. \textbf{Multi-omics identify falling LRRC15 as a COVID-19 severity marker and persistent pro-thrombotic signals in convalescence.} \textit{Nature Communications} 2022. \href{https://doi.org/10.1038/s41467-022-35454-4}{10.1038/s41467-022-35454-4}

\vspace{4pt plus 1pt}

% elife proteomics
\item \textbf{Jack S. Gisby}\textsuperscript{\textdagger}, Candice L Clarke\textsuperscript{\textdagger}, Nicholas Medjeral-Thomas\textsuperscript{\textdagger}, [...], Michelle Willicombe\textsuperscript{\textdagger}, David C Thomas\textsuperscript{\textdagger}, James E Peters\textsuperscript{\textdagger}. \textbf{Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death.} \textit{eLife} 2021. \href{https://doi.org/10.7554/eLife.64827}{10:e64827}
\vspace{4pt plus 1pt}

% transposons 
\item \textbf{Jack S. Gisby}, Marco Catoni. \textbf{The widespread nature of Pack-TYPE transposons reveals their importance for plant genome evolution.} \textit{PLOS Genetics} 2022. \href{https://doi.org/10.1371/journal.pgen.1010078}{10.1371/journal.pgen.1010078}

\textdagger Equal contributions \hfill \orcidlink{https://orcid.org/0000-0003-0511-8123}{\hspace{0.3mm} \href{{https://orcid.org/0000-0003-0511-8123}}{orcid.org/0000-0003-0511-8123}}

\end{rSection}
%----------------------- Other research works
\begin{rSection}{Other research works}

\vspace{1pt plus 1pt}

% COVID MR FAS  preprint
\item Lucija Klaric\textsuperscript{\textdagger}, \textbf{Jack S. Gisby}\textsuperscript{\textdagger}, Artemis Papadaki\textsuperscript{\textdagger}, [...], James F Wilson\textsuperscript{\textdagger}, James E Peters\textsuperscript{\textdagger}. \textbf{Mendelian randomisation identifies alternative splicing of the FAS death receptor as a mediator of severe COVID-19.} \textit{medRxiv} 2021. \href{https://doi.org/10.1101/2021.04.01.21254789 }{2021.04.01.21254789}

\textdagger Equal contributions \hfill \orcidlink{https://orcid.org/0000-0003-0511-8123}{\hspace{0.3mm} \href{{https://orcid.org/0000-0003-0511-8123}}{orcid.org/0000-0003-0511-8123}}

\end{rSection}
%----------------------- Presentations
\begin{rSection}{Conferences and presentations}

\item \textbf{Rising Scientist Day 2022 (Poster)} - Multi-omics identify LRRC15 as a COVID-19 severity predictor and persistent pro-thrombotic signals in convalescence
\vspace{3pt plus 1pt minus 1pt}

\item \textbf{Biomarkers of the Future 2021 (Presentation)} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death
\vspace{3pt plus 1pt minus 1pt}

\item \textbf{HUPO Reconnect 2021 (Poster)} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death
\vspace{3pt plus 1pt minus 1pt}

\item \textbf{UK-CIC Immunology 2021 (Poster)} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death
\vspace{3pt plus 1pt minus 1pt}

\item \textbf{Longitudinal Studies 2021, Wellcome Genome Campus (Presentation)} - Longitudinal proteomic profiling of dialysis patients with COVID-19 reveals markers of severity and predictors of death

\end{rSection}
%----------------------- Courses
\begin{rSection}{Additional Courses}

\item "From large datasets to biological insight" by Wellcome Connecting Science \& EMBL-EBI
\vspace{2pt plus 1pt minus 1pt}

\item "FAIR in (Biological) Practice" by University of Edinburgh (Ed-DaSH)
\vspace{2pt plus 1pt minus 1pt}

\item "Data Science workflows with Nextflow" by University of Edinburgh (Ed-DaSH)
\vspace{2pt plus 1pt minus 1pt}

\item "Introduction to Assessment and Feedback" by Imperial College London Graduate School
\vspace{2pt plus 1pt minus 1pt}

\item "Introduction to Teaching and Learning" by Imperial College London Graduate School
\vspace{2pt plus 1pt minus 1pt}

\item "Profiling and optimisation in Python" by Imperial College London Research Computing
\vspace{2pt plus 1pt minus 1pt}

\item "Deep Learning in the Life Sciences" by MITx
\vspace{2pt plus 1pt minus 1pt}

\item "Software Development with C++" by University of Birmingham Research Computing
\vspace{2pt plus 1pt minus 1pt}

\end{rSection}
%----------------------- References
\begin{rSection}{References}

\begin{center}
\begin{tabular}{c@{\hskip 0.75in}c} 

 Dr James Peters (PhD Supervisor) & Dr Marco Catoni (Undergraduate Project) \\ 
 Clinical Reader in Rheumatology & Lecturer in Plant Biology \\ 
 Faculty of Medicine & School of Biosciences \\ 
 Imperial College London & University of Birmingham \\ 
 Hammersmith & Edgbaston \\ 
 W12 0NN & B15 2TT \\ 
 j.peters@imperial.ac.uk & m.catoni@bham.ac.uk \\ 
\end{tabular}
\end{center}

\end{rSection}
\end{document}
